engine/src/main/resources/templates/Advisor/insightsEmailBody.html

```diff
@@ -560,7 +560,7 @@
                                 </tr>
                                 <tr>
                                     <td class="rh-masthead__subhead">
-                                        <h1><b>{#insert content-title}{/}</b></h1>
+                                        <h1><b style="line-height: 1.5;">{#insert content-title}{/}</b></h1>
```

Comment:
I didn't know `style` worked on a `b` tag :smiley:

Reply:
It seemed to work :)  But I'll change it to the h1.